config.json

```
@@ -1143,6 +1143,62 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
-  "tags": []
+  "key_features": [
+    {
+      "title": "Evolving",
+      "content": "Typescript is a modern and constantly evolving language supported by Microsoft."
+    },
+    {
+      "icon": "features-cross-platform",
+      "title": "Widely used",
+      "content": "Typescript is used by many frontend frameworks, and can be used in Javascript based ones as well."
+    },
+    {
+      "icon": "features-strongly-typed",
+      "title": "Strongly typed",
```

```suggestion
      "title": "Typed JavaScript",
```

See my TAGS.md change around "strong typing"